src/components/EditProfile.js

```diff
@@ -89,6 +89,7 @@ export class EditProfile extends Component<Props, State> {
         await this.saveProfile({
           name: this.state.name,
           bio: this.state.bio,
+          hasStake: true,
```

Comment:
Any reason to provide this here? That should be handled server-side rather than having to trust the client.